app/controllers/home_controller.rb

```diff
@@ -1,8 +1,11 @@
 class HomeController < ApplicationController
-  before_filter :authenticate_user!
-
+  # before_filter :authenticate_user!
   def index
-    @event = Event.first
+    @ip = request.remote_ip
+    @user_location = Geokit::Geocoders::MultiGeocoder.geocode("184.154.83.119")
+    @events = Event.within(3, :origin => @user_location.ll)
+    @locations = []
+    @events.each { |event| @locations << [event.lat, event.lng, event.title] }
```

Comment:
Everything looks good! I think we should find a way to associate the location with the event rather then creating a new array with the lat long and event title. It feels better to be able to have all the event attributes associated with the object as opposed to isolating the lat and long. Looking into it now...